compiler/qsc_frontend/src/compile.rs

```diff
@@ -401,6 +401,10 @@ pub fn std(store: &PackageStore, target: TargetProfile) -> CompileUnit {
                 "arithmetic_internal.qs".into(),
                 include_str!("../../../library/std/arithmetic_internal.qs").into(),
             ),
+            (
+                "arithmetic_unstable.qs".into(),
```

Comment:
Nitpick: it would be nice to have the name be "unstable_arithmetic.qs" so that it matched the order of the namespace name and so that unstable files would all be together in the alphabetical order (assuming we have more in the future).

Reply:
Renamed files containing unstable code to start with "unstable_"